src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str.cs

```diff
@@ -1,221 +1,219 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.IO;
-using System.IO.FileSystem.Tests;
 using Xunit;
 
-public class Directory_GetFileSystemEntries_str : FileSystemTest
+namespace System.IO.FileSystem.Tests
 {
-    #region Utilities
-
-    /// <summary>
-    /// Indirection around Directory.GetFileSystemEntries that child test classes may
-    /// replace with a more fitting version
-    /// </summary>
-    public virtual String[] GetEntries(String dirName)
+    public class Directory_GetFileSystemEntries_str : FileSystemTest
     {
-        return Directory.GetFileSystemEntries(dirName);
-    }
-
-    #endregion
+        #region Utilities
 
-    #region UniversalTests
-
-    [Fact]
-    public void NullFileName()
-    {
-        Assert.Throws<ArgumentNullException>(() => GetEntries(null));
-    }
-
-    [Fact]
-    public void EmptyFileName()
-    {
-        Assert.Throws<ArgumentException>(() => GetEntries(String.Empty));
-    }
+        public static string[] WhitespaceVariations = new string[] { "         ", " ", "\n", ">", "<", "\t" };
```

Comment:
Agreed. It was originally just used for whitespace, hence the name, but I later added the arrows and neglected to change it. I'll change it to something like "WindowsInvalidUnixValid" that tells you more about its purpose.